internal/support/scheduler/schedule.go

```diff
@@ -555,7 +535,7 @@ func getHttpRequest(
 		return nil, err
 	}
 
-	req.Header.Set(ContentTypeKey, ContentTypeJsonValue)
+	req.Header.Set(clients.ContentType, clients.ContentTypeJSON)
 
 	if len(params) > 0 {
 		req.Header.Set(ContentLengthKey, strconv.Itoa(len(params)))
```

Comment:
This file `internal/support/scheduler/schedule.go`  only for V1, it can be removed.

Reply:
removed